src/conversion_helper.js

```diff
@@ -870,27 +924,35 @@ function roundConversions(conversions) {
   return conversions.map(conversion => {
     function createMetricMap(imperial, metric, unit) {
       let rounded;
-      if (imperial.toString().indexOf('.') !== -1) {
-        const decimals = imperial.split('.')[1].length;
-        rounded = roundToDecimalPlaces(metric, decimals);
-
-      } else if ((imperial > 100 || metric > 100) && imperial.toString()[imperial.length - 1] == '0') {
-        rounded = round(metric, 5);
-      } else {
-        rounded = round(metric, 3);
-      }
-      return createMap(rounded, unit);
+      let result = {
+        'numbers': [],
+        'unit': unit
+      };
+
+      let index = 0;
+      imperial.forEach(function(item) {
```

Comment:
Same, I think we can use a map here. (The map function even provides you with a [free index parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map), win!)